src/state/actions/code.js

```diff
@@ -0,0 +1,21 @@
+import axios from "axios"
+import localForage from "localforage"
+import { createAction } from "redux-actions"
+import { VERIFY_FAILED, VERIFY_SUCCESSED } from "../types"
+
+const verifySuccessed = code => createAction(VERIFY_SUCCESSED)(code)
+const verifyFailed = err => createAction(VERIFY_FAILED)(err)
```

Comment:
Here you can 
```suggestion
const verifyFailed = createAction(VERIFY_FAILED)
```

This is because `createAction(type)` returns the function `payload => ({ type: type, payload: payload})`